pages/categoryPage/category.html

```diff
@@ -2,10 +2,478 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Categories</title>
     <link rel="stylesheet" type="text/css" href="./category.css">
+    <script src="https://kit.fontawesome.com/5e5573fe57.js" crossorigin="anonymous"></script>
 </head>
 <body>
+<div class="header">
+    <div class="hamburgerMenu">
```

Comment:
all class names should be: class="example-class-name"

Reply:
> all class names should be: class="example-class-name"

You can't use camelCase for classes, or kebab-case is simply just a better practice?